models/chainlink/bnb/chainlink_bnb_price_feeds.sql

```diff
@@ -4,7 +4,7 @@
     materialized = 'incremental',
     file_format = 'delta',
     incremental_strategy = 'merge',
-    unique_key = ['blockchain', 'block_time', 'proxy_address', 'oracle_price', 'aggregator_address', 'underlying_token_address'],
+    unique_key = ['blockchain', 'block_number', 'proxy_address', 'aggregator_address', 'underlying_token_address'],
```

Comment:
Hi @0xroll , in my last commit I change the unique_key list to 5 elements, please have a check if this still works.